docs/2.0/integration/testng.adoc

```diff
@@ -1,8 +1,8 @@
 = TestNG
 An Allure report can be generated for any TestNG test. In order to get test results, you need to:
 
-* Download and uzip the latest link:https://dl.bintray.com/qameta/generic/io/qameta/allure/allure/2.0-BETA6/allure-2.0-BETA6.zip[allure-commandline]
-* Add **allure-java** dependency. Note, you may need to build link:https://github.com/allure-framework/allure-java[allure-java] manually to get a SNAPSHOT version of the artefact
+* Download and uzip the latest link:https://dl.bintray.com/qameta/generic/io/qameta/allure/allure/2.0.1/allure-2.0.1.zip[allure-commandline]. Or you can use the following link:https://github.com/qameta/homebrew-allure[brew formula] to simplify the process.
```

Comment:
May work, if we update those section first. As command-line link for Windows should refer to bintray instead of project releases I believe + beta versions should be cleaned up as well.